src/Type/Data/Peano/Int/Definition.purs

```diff
@@ -27,25 +28,23 @@ foreign import data Pos :: Nat -> Int
 -- |
 foreign import data Neg :: Nat -> Int
 
-data IProxy (i :: Int) = IProxy
-
 class IsInt (i :: Int) where
   -- | reflect a type-level Int to a value-level Int
   -- |
   -- | ```purescript
-  -- | reflectInt (undefined :: IProxy N10) = -10
+  -- | reflectInt (undefined :: y N10) = -10
   -- | ```
   -- |
-  reflectInt :: IProxy i -> Int
+  reflectInt :: Proxy i -> Prim.Int
```

Comment:
Did you mean to write this instead? `forall proxy. proxy i -> Int`. Then it will work for both `Proxy` and `IProxy` types. See [this comment](https://github.com/purescript/purescript-prelude/commit/11004596cbaaa7f09274fa4c8554e1c0631755a6) for more details on why we did this.

Reply:
I've removed the `IProxy`/`NProxy` kinds from the project in favor of `Proxy` which I've meant to be a breaking change

Reply:
I think following Jordans advice still makes sense. It would  mean that the library would compile on purescript 0.13.* **and** 0.14

Reply:
I see, although I can't say I'm quite sure what the implications of targeting 0.13.x as well would be; maybe resurrecting `IProxy`/`NProxy`, and then generalizing the types?

Reply:
It's not really to target `0.13.x`. It's to make one's code break less when they upgrade to `v0.14.0`. Using `forall proxy. proxy i -> Int` means I can continue to use `IProxy` rather than `Proxy` when I first upgrade as there will be many other breaking changes to account for. At some point in the future, I should change `IProxy` to `Proxy`. Then, we'll release `v0.15.0`, which will remove all the kind-specific `Proxy` types (e.g. `IProxy`), so that one's only choice is `Proxy`.

Reply:
That seems fair, much appreciated